ports/atmel-samd/boards/circuitplayground_express/mpconfigboard.h

```diff
@@ -8,52 +8,25 @@
 #define SPI_FLASH_BAUDRATE  (8000000)
 
 // On-board flash
-#define SPI_FLASH_MOSI_PIN          PIN_PA20
-#define SPI_FLASH_MISO_PIN          PIN_PA16
-#define SPI_FLASH_SCK_PIN           PIN_PA21
-#define SPI_FLASH_CS_PIN            PIN_PB22
-
-#define SPI_FLASH_MOSI_PIN_FUNCTION PINMUX_PA20D_SERCOM3_PAD2
-#define SPI_FLASH_MISO_PIN_FUNCTION PINMUX_PA16D_SERCOM3_PAD0
-#define SPI_FLASH_SCK_PIN_FUNCTION  PINMUX_PA21D_SERCOM3_PAD3
-#define SPI_FLASH_SERCOM            SERCOM3
-#define SPI_FLASH_SERCOM_INDEX      3
-#define SPI_FLASH_MOSI_PAD          2
-#define SPI_FLASH_MISO_PAD          0
-#define SPI_FLASH_SCK_PAD           3
-
-// <o> Transmit Data Pinout
-// <0x0=>PAD[0,1]_DO_SCK
-// <0x1=>PAD[2,3]_DO_SCK
-// <0x2=>PAD[3,1]_DO_SCK
-// <0x3=>PAD[0,3]_DO_SCK
-#define SPI_FLASH_DOPO              1
-#define SPI_FLASH_DIPO              0    // same as MISO PAD
+#define SPI_FLASH_MOSI_PIN          &pin_PA20
+#define SPI_FLASH_MISO_PIN          &pin_PA16
+#define SPI_FLASH_SCK_PIN           &pin_PA21
+#define SPI_FLASH_CS_PIN            &pin_PB22
 
 // These are pins not to reset.
 // PA24 and PA25 are USB.
```

Comment:
Remove that comment for this board, but maybe in all the board files, add a comment that says USB (PA24 and PA25), and SWD (PA30 and PA31) are protected from reset elsewhere.

Reply:
I removed the comment but don't think we need a comment in each. Boards shouldn't need to worry about USB or SWD. I want to move away from this mask approach to using never reset anyway. Just didn't do everything because this PR has a lot already.